### PR TITLE
Find sh.exe when Git 2.5 for windows is installed

### LINKED ISSUE #112: Docker Toolbox Migration of Boot2Docker VM failed on Windows 7, Git not found
Per the installer instructions (and because I did not see another instance of this one), logging a new issue. 
image

It appears the path to Git is hard coded somewhere in the install script. Here is toolbox-migration-logs.txt:

    C:\Program Files\Docker Toolbox>cd C:\Program Files\Docker Toolbox\ 

    C:\Program Files\Docker Toolbox>"C:\Program Files (x86)\Git\bin\sh.exe" --login -i migrate.sh 
    The system cannot find the path specified.

My Git is installed in a separate folder.

Subsequently running `>docker-machine create -d virtualbox --virtualbox-import-boot2docker-vm boot2docker-vm dev` worked just fine so not a critical defect but something to investigate.

### PR DESCRIPTION
**Pull request by afscrome**
Fixes #112

### CODE REVIEW

**Comment by mchiang0610** [comment]
@afscrome Thanks for submitting this. We will begin bundling Git 2.5, and will test this with it before release. 

Thanks!

File: windows/migrate.bat [hunk 1]
```diff
@@ -1,4 +1,13 @@
...
 cd %~dp0
-"C:\Program Files (x86)\Git\bin\sh.exe" --login -i migrate.sh
+
+:: Add known locations of Git to path to try and find sh.exe
+set PATH=%PATH%;%ProgramFiles%\git\bin\;%ProgramFiles(x86)%\Git\bin\
+
+where sh.exe 
+if ERRORLEVEL 1 (
+    echo "Could not find sh.exe"
+    exit 1234
```
Comment (jeffdm) [inline windows/migrate.bat]: Maybe just exit 1?

**Comment by jeffdm** [comment]
One comment and then LGTM

**Comment by jeffdm** [comment]
Actually - I'll fix this on master since it's a tiny change. LGTM

**Comment by jeffdm** [comment]
Can you sign your commit? For ex:
```
$ git commit --amend -s --no-edit
$ git push -f
```

**Comment by jeffdm** [comment]
This should have a fix on master to use the 64 bit Git 2.5 on Windows (trying to get this fix out very fast since 2.5 is out for 1+ weeks). I'll close this for now but a huge thanks for the PR!

**Pull request closed without merging**